### DISCUSSION TITLE
Integrate Excalidraw component in angular project

### DISCUSSION THREAD

**Discussion by Shivugethub** [excalidraw/excalidraw#4100] [Q&A] [answered]
Hi excalidraw team could you please help me to sort out this issue, working hard on it to get integrated in angular, it could be great  
if you guys provide a sample angular excalidraw integration or guide me in the below provided sandbox code. 

sample code link in codesandbox - https://codesandbox.io/s/xtjyq?file=/src/app/ngexcalidraw.tsx 

![image](https://user-images.githubusercontent.com/33868554/138599178-189423e4-529a-4c05-8971-212d01d57166.png)

Thank you.

**Comment by ad1992** [comment, marked as answer]
@Shivugethub the example you shared is giving some error from the server, can you fix that so I can check the compile errors you mentioned ?
Also since the App is using angular, instead of bundling react, react-dom to your main bundle you might want to use directly in browser as mentioned in [usage in browser](https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#in-browser) and you don't have to worry about web bundler as well

**Reply by Shivugethub** [reply]
@ad1992, it's working fine via in browser (CDN)  Thank you

**Reply by ad1992** [reply]
Awesome 👍🏻

**Reply by rIGMORSOMRAR** [reply]
Hi, @Shivugethub, can you let me know how to integrate it with the Angular CLI Project?

**Reply by Shivugethub** [reply]
Hi, refer this it might help you https://github.com/excalidraw/excalidraw/issues/4013#issuecomment-980640397
